packages/@misk/cli/src/utils/miskTabUtilities.ts

```diff
@@ -49,11 +49,9 @@ export const readMiskTabJson = (dir: string): IMiskTabJSON => {
   try {
     return fs.readJSONSync(makePath(dir, Files.miskTab))
   } catch (e) {
-    console.log(
-      `[FATAL] Failed to read ${dir}/${Files.miskTab}. Try this command in a tab directory.\n\n`,
-      e
+    throw Error(
```

Comment:
If an error is thrown, do we still need the return below?

Reply:
The error is sufficient to cause an exit because it is uncaught in the CLI